cli/_paths.js

```diff
@@ -0,0 +1,38 @@
+// @ts-check
```

Comment:
`_path.js` sounds like the file is not being used.
I find it a little weird that it has an `_` prefix.

Reply:
It's my old habit to change the sort order. I wanted to make it look distinct from the bin entries in the same `cli` path. Or we can create a nested folder for it.

Reply:
moved to `#cli/*` -> `cli/common/*`